_posts/2015-03-31-how-we-do-css.md

```diff
@@ -0,0 +1,173 @@
+---
+layout: post
+title: How We Do CSS at Zumba
+description: This is how we do CSS at Zumba.
+tags: [technology, css, sass, front end, engineering]
+author: neptunz
+---
+
+As you may or may not know Zumba's website consists of a robust app that includes a fully custom ecommerce shop that supports international transactions, class and training search and registration, Zumba Instructor Network (ZIN) admin portal and many other apps. Our in-house tech department is responsible for managing the interactive products, developing, testing and deploying the projects.
+
+I was inspired by the teams at <a href="http://dev.ghost.org/css-at-ghost/" target="_blank">Ghost</a> and <a href="https://github.com/styleguide/css" target="_blank">Github</a> to write this article to give you an idea of how we do CSS at Zumba.
+
+
+### Intro - Browser and Device Support
+We try to keep our code as light, modular and DRY as possible. We use Modernizr, Autoprefixer and some Foundation components. We try to keep our nesting no more than 4 levels deep and we try to make sure that  our naming convention lends itself to reusable components.
```

Comment:
You repeat "we try" here 4 times in 2 sentences. Replace it with something else like: 
"We follow guidelines to keep our nesting no more than 4 levels deep"

"We try" sounds very wishy washy, you should put something more assertive here anyway.